tests/extrusion/test_2d_cohomology.py

```diff
@@ -113,8 +113,8 @@ def test_betti1(horiz_complex, vert_complex):
 
     W0_elt = TensorProductElement(U0, V0)
 
-    W1_a = HDiv(TensorProductElement(U1, V0))
-    W1_b = HDiv(TensorProductElement(U0, V1))
+    W1_a = HDivElement(TensorProductElement(U1, V0))
```

Comment:
This is a breaking public API change. We can't do this to users without warning.

Reply:
I think I've sucessfully reverted this change. I was getting issues somewhere due to `ufl.HDiv` being the Sobolev space, but I'm not sure why I changed all these rather than making sure the function `HDiv` was defined/imported later

Reply:
Nevermind, I see the issue: You used to be able to call `ufl.HDiv` but that got removed in https://github.com/FEniCS/ufl/pull/197 when we deprecated `HDivElement`. Update to finat incoming...

Reply:
I think https://github.com/FInAT/FInAT/pull/111/commits/666e924899c063254655e8b54bf937a63a715b6e should restore the old behaviour of HDiv and HCurl

Reply:
I've opened an issue in case you want to remove my hack to keep this syntax at some point: https://github.com/firedrakeproject/firedrake/issues/3213